app/auth/screens/SignInScreen.tsx

```diff
@@ -0,0 +1,88 @@
+import { useState } from 'react';
+import { View, Text, TextInput, Pressable, ScrollView } from 'react-native';
+import { Link } from 'expo-router';
+import { useAuth } from '../context/AuthContext';
+import { Redirect } from 'expo-router';
+import { randomDirector } from '../../shared/utils/randomDirector';
+
+export default function SignInScreen() {
+  const { signIn, user, isLoading } = useAuth();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
```

Comment:
Can we use `string | null` as possible types instead of `string` combined with empty string?